docs/Release-Process.md

```diff
@@ -0,0 +1,36 @@
+# Release process for Swarm
+
+This document describes the steps required to perform a new release.
+
+## Pre release
+
+1. Make sure that the most recent builds are green and that smoke tests are passing on the cluster.
+2. Check if protocols should be bumped. (e.g [PR#1465](https://github.com/ethersphere/swarm/pull/1465))
```

Comment:
This is also related to verifying that the commit we want to release is healthy.

Probably best to split the `Release process` document from a document describing how we test and what we consider stable release candidate.